yellowbrick/model_selection/CVScores.py

```diff
@@ -0,0 +1,174 @@
+
+# coding: utf-8
+
+# In[ ]:
+
+# yellowbrick.model_selection.CVScores
+#
+#
+# Author:   Prema Damodaran Roman
+
+#
+# Copyright (C) 2018 District Data Labs
+# For license information, see LICENSE.txt
+#
+# ID: CVScores.py [7f47800] pdamo24@gmail.com $
+
+##########################################################################
+## Imports
+##########################################################################
+
+import warnings
+import numpy as np
+import matplotlib as mpl
```

Comment:
These unused imports are the second place the test failed:

```
/home/travis/build/DistrictDataLabs/yellowbrick/yellowbrick/model_selection/CVScores.py:21: UnusedImport
'warnings' imported but unused
/home/travis/build/DistrictDataLabs/yellowbrick/yellowbrick/model_selection/CVScores.py:23: UnusedImport
'matplotlib as mpl' imported but unused
```

Would you mind removing these two imports (`import warnings` and `import matplotlib as mpl`)?